.circleci/config.yml

```diff
@@ -10,54 +12,36 @@ defaults: &defaults
 # restore_repo: &restore_repo
 #   restore_cache:
 #     key: v1-repo-{{ .Branch }}-{{ .Revision }}
-install_js: &install_js
-  run:
-    name: Install js dependencies
-    command: |
-      yarn config set yarn-offline-mirror ~/.cache/npm-packages-offline-cache/v1
-      yarn
-restore_yarn_cache: &restore_yarn_cache
-  restore_cache:
-    key: v2-yarn-sha-{{ checksum "yarn.lock" }}
-restore_yarn_offline_mirror: &restore_yarn_offline_mirror
-  restore_cache:
-    key: v2-npm-packages-offline-cache
-version: 2
+
+commands:
+  install_js:
+    steps:
+      - restore_cache:
+          keys:
+             - v2-yarn-sha-{{ checksum "yarn.lock" }}
+             - v2-yarn-sha-
```

Comment:
Should we add a comment about the partial cache restoration?

Reply:
What should we include there? I don't think it's useful to repeat documentation in code comments.

Reply:
I had to dig into CircleCI docs to understand what was going on. People will use our configuration as a reference.

Reply:
> dig into CircleCI

You mean google "CircleCI cache"? 

Could you answer what that comment would say? "Partial cache restoration" would be misleading.